components/automate-ui/src/app/pages/+compliance/shared/reporting/stats.service.ts

```diff
@@ -277,26 +277,13 @@ export class StatsService {
     }, []);
 
     if (reportQuery.startDate) {
-      const now = moment();
-      const value = reportQuery.startDate
-        .set({
-          hour: now.get('hour'),
-          minute: now.get('minute'),
-          second: now.get('second')
-        }).utc().startOf('day');
+      const value = reportQuery.startDate.clone().utc().startOf('day');
 
       apiFilters.push({type: 'start_time', values: [value.format()]});
     }
 
     if (reportQuery.endDate) {
-      const now = moment();
-      const value = reportQuery.endDate
-        .set({
-          hour: now.get('hour'),
-          minute: now.get('minute'),
-          second: now.get('second')
-        }).utc().endOf('day');
-
+      const value = reportQuery.endDate.clone().utc().endOf('day');
```

Comment:
Pretty sure you do not need the `clone()` there as you are evaluating a functional sequence.

Reply:
That is what I thought from my original PR, but the `endOf('day')` function changes the original object. https://momentjs.com/docs/#/manipulating/end-of/